### DISCUSSION TITLE
Im trying to create an issue but get sent to github email settings...

### DISCUSSION THREAD

**Discussion by sugoidesune** [vuetifyjs/vuetify#11417] [General] [answered]
Im trying to create an issue with the vuetify issue creator but get sent to github email settings... and the issue is not created.

**Comment by nekosaur** [comment, marked as answer]
Hm. We had someone else that recently had the same problem. I thought [this](https://github.com/vuetifyjs/vuetify-issue-helper/pull/47) was supposed to fix it. Could you try hard reloading issues.vuetifyjs.com and see if it works?

Ping @MajesticPotatoe

**Reply by sugoidesune** [reply]
I tried again today with hard reloading and it worked.

**Comment by nekosaur** [comment]
@sugoidesune Are you still having trouble with the issue creator?
